interface/modules/custom_modules/oe-module-comlink-telehealth/src/TelehealthGlobalConfig.php

```diff
@@ -0,0 +1,182 @@
+     * @return bool
+     */
+    public function isTelehealthConfigured()
+    {
+        $config = $this->getGlobalSettingSectionConfiguration();
+        $keys = array_keys($config);
+        foreach ($keys as $key) {
+            if ($key == $this->isOptionalSetting($key)) {
+                continue;
+            }
+            $value = $this->getGlobalSetting($key);
+
+            if (empty($value)) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public function getInstitutionId()
+    {
+        return $this->getGlobalSetting(self::UNIQUE_INSTALLATION_ID);
+    }
+
+    public function getInstitutionName()
+    {
+        return $this->getGlobalSetting(self::INSTALLATION_NAME);
+    }
+
+    public function getRegistrationAPIURI()
+    {
+        return $this->getGlobalSetting(self::COMLINK_VIDEO_REGISTRATION_API);
+    }
+
+    public function getTelehealthAPIURI()
+    {
+        return $this->getGlobalSetting(self::COMLINK_VIDEO_TELEHEALTH_API);
+    }
+
+    public function getRegistrationAPIUserId()
+    {
+        return $this->getGlobalSetting(self::COMLINK_VIDEO_API_USER_ID);
+    }
+
+    public function getRegistrationAPIPassword()
+    {
+        $encryptedValue = $this->getGlobalSetting(self::COMLINK_VIDEO_API_USER_PASSWORD);
+        return $this->cryptoGen->decryptStandard($encryptedValue);
+    }
+
+    public function getRegistrationAPICmsId()
+    {
+        return $this->getGlobalSetting(self::COMLINK_VIDEO_TELEHEALTH_CMS_ID);
+    }
+
+    public function shouldAutoProvisionProviders(): bool
+    {
+        $setting = $this->getGlobalSetting(self::COMLINK_AUTO_PROVISION_PROVIDER);
+        return $setting !== "";
+    }
+
+    public function getGlobalSetting($settingKey)
+    {
+        global $GLOBALS;
+        // don't like this as php 8.1 requires this but OpenEMR works with globals and this is annoying.
+        return $GLOBALS[$settingKey] ?? null;
+    }
+
+    public function getAppRegistrationCodeLength()
+    {
+        return self::APP_REGISTRATION_CODE_LENGTH;
+    }
+
+    public function getGlobalSettingSectionConfiguration()
+    {
+        $settings = [
+            self::COMLINK_VIDEO_REGISTRATION_API => [
+                'title' => 'Telehealth Registration URI'
+                ,'description' => 'Registration endpoint URI'
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+            ]
+            ,self::COMLINK_VIDEO_TELEHEALTH_API => [
+                'title' => 'Telehealth Video API URI'
+                ,'description' => 'The URI for the video bridge api'
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+            ]
+            ,self::COMLINK_VIDEO_API_USER_ID => [
+                'title' => 'Telehealth Installation User ID'
+                ,'description' => 'This is your unique video application api user id.  Contact ComLink if you have not received it'
```

Comment:
remove the double spacing in this string and others (they throw off the translation engine)

Reply:
fixed